test/ui/browser/actions/test-set-page-order.js

```diff
@@ -1,14 +1,90 @@
 // Any copyright is dedicated to the Public Domain.
 // http://creativecommons.org/publicdomain/zero/1.0/
 
-import expect from 'expect'; // eslint-disable-line
+import expect from 'expect';
 import configureStore from '../../../../ui/browser/store/store';
-import * as actions from '../../../../ui/browser/actions/main-actions'; // eslint-disable-line
+import * as actions from '../../../../ui/browser/actions/main-actions';
+
+/**
```

Comment:
This might be simpler if you used tab titles, rather than extracting things from the URLs.